internal/adapter/graphql/convert.go

```diff
@@ -70,11 +70,25 @@ func toWidgetLayout(w *scene.WidgetLayout) *WidgetLayout {
 			Area:    (*WidgetAreaType)(&w.DefaultLocation.Area),
 		}
 	}
-	wl := WidgetLayout{
-		Extendable:      w.Extendable,
-		Extended:        w.Extended,
-		Floating:        w.Floating,
-		DefaultLocation: l,
+	var e *WidgetExtendable
+	var wl WidgetLayout
+	if w.Extendable != nil {
+		e = &WidgetExtendable{
+			Vertically:   w.Extendable.Vertically,
+			Horizontally: w.Extendable.Horizontally,
+		}
+		wl = WidgetLayout{
+			Extendable:      e,
+			Extended:        w.Extended,
+			Floating:        w.Floating,
+			DefaultLocation: l,
+		}
```

Comment:
Very strange. The reason I didn't do how you suggest was because I was getting a `nil pointer` type panic.
Now when I try it is working. I wonder why..